frontend/src/scenes/notebooks/Notebook/notebooksListLogic.ts

```diff
@@ -44,16 +48,30 @@ export const openNotebook = (
     }
 
     popoverLogic?.actions.setInitialAutofocus(focus)
+
+    const theNotebookLogic = notebookLogic({ shortId: notebookId })
+    const unmount = theNotebookLogic.mount()
+
+    await theNotebookLogic.asyncActions.editorIsReady()
```

Comment:
Whaaaaaaat. What is an async action? In my head it would be a helper that awaits all async listeners of an action? But here it seems to be that that action is never called? I'm so confused :D

Reply:
Also could we get in a weird situation where the unmount is never called if an error occurs? Do we need to wrap it with a try finally or something? So many questions 😅

Reply:
🤷 😅